strictdoc/export/html/html_generator.py

```diff
@@ -487,6 +493,25 @@ def export_project_tree_screen(
         with open(output_file, "w", encoding="utf8") as file:
             file.write(output)
 
+    def export_project_map(
+        self,
+        *,
+        traceability_index: TraceabilityIndex,
+    ):
+        _assets_dir = os.path.join(
```

Comment:
You probably don't need `_` here. Let's keep it at just `assets_dir`. Otherwise, it looks too unusual.